spec/facets/StorageDatasetFacet.json

```diff
@@ -17,12 +17,13 @@
             "fileFormat": {
               "description": "File format with allowed values: parquet, orc, avro, json, csv, text, xml.",
               "type": "string"
+            },
+            "physicalLocation": {
+              "description": "Physical location of the stored dataset.",
```

Comment:
Do you have an example? Is it meant to be a url? I think we should make the description more specific of what is a valid value for this field.

Reply:
I think naming for location should be consistent with how we'd name `dataset` pointing to this location. 

So, instead of for example `s3://bucket/directory`, we should have `{"namespace": "s3://bucket", "name": "directory"}`.

Reply:
@mobuchowski great idea, let's do that this way.